documentation/modules/ref-kafka-user.adoc

```diff
@@ -41,12 +45,12 @@ spec:
   # ...
 ----
 
-When the user is created by the User Operator, it will create a new secret with the same name as the `KafkaUser` resource.
-The secret will contain a public and private key which should be used for the TLS Client Authentication.
-Bundled with them will be the public key of the client certification authority which was used to sign the user certificate.
-All keys will be in X509 format.
+When the user is created by the User Operator, it creates a new secret with the same name as the `KafkaUser` resource.
+The secret contains a public and private key to use for TLS client authentication.
+Bundled with the keys is the public key of the client Certificate Authority (CA) used to sign the user certificate.
```

Comment:
you mention the "user certificate" here which is actually the "public key" mentioned in the line above.
We should always remember that a certificate contains a public key signed by another certificate coming from a CA.
So the user certificate is/contains actually the public key which is signed by the client CA certificate (which is itself a public key but self-signed in our case).

Reply:
I'm hoping this is clearer:

> The public key is contained in a user certificate, which is signed by the client Certificate Authority (CA).

Reply:
it's better yes.